packages/ui/package.json

```diff
@@ -58,29 +58,29 @@
   },
   "dependencies": {
     "@fontsource-variable/inter": "5.0.1",
-    "@headlessui/react": "1.7.7",
+    "@headlessui/react": "2.2.0",
     "@heroicons/react": "2.0.13",
-    "@radix-ui/react-accordion": "^1.1.2",
-    "@radix-ui/react-avatar": "^1.0.3",
-    "@radix-ui/react-checkbox": "1.0.4",
-    "@radix-ui/react-context-menu": "2.1.4",
-    "@radix-ui/react-dialog": "1.0.4",
-    "@radix-ui/react-dropdown-menu": "2.0.5",
-    "@radix-ui/react-hover-card": "^1.0.6",
-    "@radix-ui/react-icons": "^1.3.0",
-    "@radix-ui/react-label": "2.0.2",
-    "@radix-ui/react-navigation-menu": "1.1.3",
-    "@radix-ui/react-popover": "1.0.6",
-    "@radix-ui/react-progress": "1.0.3",
-    "@radix-ui/react-scroll-area": "1.0.4",
-    "@radix-ui/react-select": "1.2.2",
-    "@radix-ui/react-separator": "1.0.3",
-    "@radix-ui/react-slider": "^1.1.2",
-    "@radix-ui/react-slot": "1.0.2",
-    "@radix-ui/react-switch": "1.0.3",
-    "@radix-ui/react-tabs": "^1.0.4",
-    "@radix-ui/react-toggle": "1.0.3",
-    "@radix-ui/react-tooltip": "1.0.6",
+    "@radix-ui/react-accordion": "1.2.2",
+    "@radix-ui/react-avatar": "^1.1.2",
```

Comment:
can we lock these various radix ui dependencies?

Reply:
yup, not sure why that happened